hello-world/scripts/deploy.js

```diff
@@ -20,7 +20,8 @@ async function main() {
 
   const HelloWorld = await ethers.getContractFactory("HelloWorld");
   const instance = await HelloWorld.deploy({
-    gasPrice: ethParams.txGasPrice
+    gasPrice: ethParams.txGasPrice,
+    gasLimit: ethParams.txGasLimit,
```

Comment:
We need to figure out how to be able to run the tests on Hardhat network with this option. Because this is a breaking change (concerning the workflow) and it is important to present both ways on how to develop with Hardhat.

Let's not merge until I have a tested out solution ready (will start working on it now).

Reply:
sounds good, do you mean we need to figure out some code that (without any modification) works on both mandala and hardhat?

Reply:
sure we can wait until that is done, no need to hurry